### ISSUE TITLE
Kirby Enhanced Textarea

### ISSUE THREAD

**Issue by aoimedia** [jenstornell/kirby-plugins#252]
**GitHub** https://github.com/medienbaecker/kirby-enhanced-textarea

> Text area field with enhanced formatting options.

![](https://raw.githubusercontent.com/medienbaecker/kirby-enhanced-textarea/master/preview.gif)

**Comment by jenstornell** [comment]
Screenshot bug it seems. Looks like an awesome plugin. I've missed the headline buttons since Kirby 1.

**Comment by aoimedia** [comment]
Looks like Thomas has updated the gifs since I opened the issue. Now fixed. Agreed. Looks like a very practical plugin. :)

**Comment by jenstornell** [comment]
Maybe it's safer to save the gif locally and then drag and drop it in? I've started to to it that way to prevent broken links.

**Comment by aoimedia** [comment]
Good point. Though this does mean if original gifs are updated we wont have the latest screenshot/animations to go with the plugin. However, I agree that this is still preferable to a broken link, so I'll do this with all future issues. 👍🏻

**Comment by jenstornell** [comment]
Yes, I think it's better to have an outdated gif than a broken one. An old one can still give a clew about what it's all about.

**Comment by aoimedia** [comment]
I agree completely.

**Comment by aoimedia** [comment]
I've gone over all my submitted/open issues and replaced gifs (now uploaded directly to GitHub cloud)… because the gods of technology abhor a broken URL. ;)
